workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -135,5 +134,14 @@ private static DownloadResult Download()
 
             return exitCode == 0 ? DownloadResult.Success : DownloadResult.Error;
         }
+
+        internal static string[] ConstructArguments()
+        {
+            var baseArgs = new List<string> { "run", "-p", $"\"{ProjectPath}\"", "--", $"\"{Common.SpatialBinary}\"", $"\"{Common.CoreSdkVersion}\"" };
+            var toolsConfig = ScriptableGdkToolsConfiguration.GetOrCreateInstance();
```

Comment:
Move `toolsConfig` to the top and then add everything to `baseArgs` all in one go.